modules/elasticsearch/src/test/java/org/testcontainers/elasticsearch/ElasticsearchContainerTest.java

```diff
@@ -84,17 +94,36 @@ public void elasticsearchOssImage() throws IOException {
         }
     }
 
+    @Test
+    public void transportClientClusterHealth() {
+        try (ElasticsearchContainer container = new ElasticsearchContainer()) {
+            container.start();
+
+            TcpHost tcpHost = container.getTcpHost();
+            TransportAddress transportAddress = new TransportAddress(new InetSocketAddress(tcpHost.getHostname(), tcpHost.getPort()));
+            String expectedClusterName = "docker-cluster";
+            Settings settings = Settings.builder().put("cluster.name", expectedClusterName).build();
+            try (TransportClient transportClient = new PreBuiltTransportClient(settings)
+                .addTransportAddress(transportAddress)) {
+                ClusterHealthResponse healths = transportClient.admin().cluster().prepareHealth().get();
+                String clusterName = healths.getClusterName();
+                assertThat("Wrong cluster name", CoreMatchers.equalTo(clusterName).matches(expectedClusterName));
```

Comment:
But I'd write something like:

```suggestion
                assertThat(clusterName, is(expectedClusterName));
```